paymentsheet/src/test/java/com/stripe/android/paymentsheet/VerticalModePaymentSheetActivityTest.kt

```diff
@@ -320,4 +358,38 @@ internal class VerticalModePaymentSheetActivityTest {
             response.setResponseCode(500)
         }
     }
+
+    private fun setupPaymentMethodUpdateResponse(paymentMethodId: String, cardBrand: String) {
+        networkRule.enqueue(
+            host("api.stripe.com"),
+            method("POST"),
+            path("/v1/payment_methods/$paymentMethodId"),
+            bodyPart(urlEncode("card[networks][preferred]"), cardBrand)
+        ) { response ->
+            val paymentMethodDetails = paymentMethodDetailsMap.values.first { it.id == paymentMethodId }
+            val originalCard = PaymentMethodFactory.card(
+                id = paymentMethodDetails.id
+            ).update(
+                last4 = paymentMethodDetails.last4,
+                addCbcNetworks = true,
+            )
+
+            val card = originalCard.copy(
+                card = originalCard.card!!.copy(
+                    displayBrand = cardBrand
+                )
+            )
+            response.setBody(PaymentMethodFactory.convertCardToJson(card).toString(2))
+        }
+    }
+
+    private data class PaymentMethodDetails(
+        val id: String,
+        val last4: String,
+    )
+
+    private val paymentMethodDetailsMap = mapOf(
```

Comment:
@amk-stripe I'll follow up this change to pass in a varargs list of these instead of using this map.

Reply:
Or I can do it here if you prefer.

Reply:
thanks!